http_service/bugbug_http/app.py

```diff
@@ -202,11 +221,7 @@ def schedule_bug_classification(model_name: str, bug_ids: Sequence[int]) -> None
 
     redis_conn.mset(job_id_mapping)
 
-    schedule_job(
-        JobInfo(classify_bug, model_name, bug_ids, BUGZILLA_TOKEN),
-        job_id=job_id,
-        timeout=BUGZILLA_JOB_TIMEOUT,
-    )
+    return job_id
```

Comment:
You can do something like: `return JobInfo(classify_bug, model_name, bug_ids, BUGZILLA_TOKEN), job_id, BUGZILLA_JOB_TIMEOUT`. This way you won't need to repeat them in the callers.

Reply:
implemented the suggestion